frameworks/helloworld/tests/test_sanity.py

```diff
@@ -21,19 +23,69 @@
 @pytest.fixture(scope="module", autouse=True)
 def configure_package(configure_security):
     try:
+        service_options = {"service": {"name": foldered_name}}
         sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
         sdk_upgrade.test_upgrade(
             config.PACKAGE_NAME,
             foldered_name,
             config.DEFAULT_TASK_COUNT,
-            additional_options={"service": {"name": foldered_name}},
+            additional_options=service_options,
         )
 
-        yield  # let the test session execute
+        yield {"package_name": config.PACKAGE_NAME, **service_options}
     finally:
         sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
 
 
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_install():
+    config.check_running(sdk_utils.get_foldered_name(config.SERVICE_NAME))
+
+
+@pytest.mark.sanity
+@pytest.mark.metrics
```

Comment:
Any reason for retaining this `metrics` mark ? Once this PR is merged, `dcos-enterprise` smoke test would run these tests and that should be good enough, am i missing something on the usage of `metrics` mark ?!

Reply:
I took the `metrics` mark from the other tests, but I agree that it's not required. Will remove it.